JeMPI_Apps/JeMPI_Controller/src/main/java/org/jembi/jempi/controller/PsqlNotifications.java

```diff
@@ -30,9 +31,9 @@ void insert(
 
          // Set auto-commit to false
          psqlClient.setAutoCommit(false);
-         Date res = new Date(created);
-         String state = "New";
-
+         Date date = new Date();
+         Timestamp res = new Timestamp(date.getTime());
+         String state = "OPEN";
          String sql = "INSERT INTO notification (id, type, state, names, created, patient_id, golden_id, score) "
```

Comment:
Can we not use PrepareStatement for this? (helps with security/SQL injection prevention)

Reply:
@walisc refactored. Thank you for the catch.